mango_v4.json

```diff
@@ -7142,7 +7266,8 @@
         "kind": "enum",
         "variants": [
           {
-            "name": "Market"
+            "name": "Market",
+            "fields": []
```

Comment:
@ckamm this change is surprising, i ran the following commands to update this file:

```
cargo run --bin anchor build -- --features enable-gpl
./idl-fixup.sh 
cp target/idl/mango_v4.json .
```

Reply:
I see the same change when I run `update-local-idl.sh`. 

When doing that, there are some `yarn tsc` because of the selfTradeBehavior addition to `perpPlaceOrderPegged`, that also needs a v2 instruction.